examples/CMakeLists.txt.in

```diff
@@ -30,7 +30,7 @@ set(DNNL_GPU_RUNTIME "@DNNL_GPU_RUNTIME@")
 
 set(DNNL_INSTALL_MODE "@DNNL_INSTALL_MODE@")
 set(IS_NEW_DIR_LAYOUT FALSE)
-if(DNNL_INSTALL_MODE STREQUAL "BUNDLE_V2")
+if(DNNL_INSTALL_MODE STREQUAL "BUNDLE")
```

Comment:
CMakeLists.txt.in is only used in the bundle mode so we can safely remove `IS_NEW_DIR_LAYOUT` and keep only the code path that is selected when `IS_NEW_DIR_LAYOUT` is `TRUE`.